src/test/java/oop/Exercise01Test/StudentControllerTest.java

```diff
@@ -0,0 +1,103 @@
+package oop.Exercise01Test;
+
+import oop.Exercise01.Student;
+import oop.Exercise01.StudentBuilder;
+import oop.Exercise01.StudentController;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class StudentControllerTest {
+
+    @Test
+    public void test_addStudent() {
+        List<Student> list = new ArrayList<Student>();
+        StudentBuilder builder = new StudentBuilder().id("01").firstName("Hue").lastName("Nguyen").birthday(new Date()).className("A1").address("Quang Nam");
+        Student student = builder.build();
+        student.toString();
+        list.add(student);
+        assertEquals(new StudentController().addStudent(student), list);
+    }
+
+    @Test
+    public void test_deleteStudent() {
+        StudentController students = new StudentController();
+        Student student = new Student(new StudentBuilder().id("01").firstName("Hue").lastName("Nguyen").birthday(new Date()).className("A1").address("Quang Nam"));
+        students.addStudent(student);
+
+        List<Student> list = new ArrayList<Student>();
+        list.add(student);
+        list.remove(student);
+
+        assertEquals(students.deleteStudent(student), list);
+    }
+
+    @Test
+    public void test_saveToFile() {
+        Student student = new Student(new StudentBuilder().id("01").firstName("Hue").lastName("Nguyen").birthday(new Date()).className("A1").address("Quang Nam"));
```

Comment:
Lệnh dài quá thì nên xuống dòng cho dễ đọc
```suggestion
        Student student = new StudentBuilder()
                                                   .id("01")
                                                   .firstName("Hue")
                                                   .lastName("Nguyen")
                                                   .birthday(new Date())
                                                   .className("A1")
                                                   .address("Quang Nam")
                                                   .build();
```

Reply:
Done